src/net/interface.go

```diff
@@ -226,7 +227,7 @@ func (zc *ipv6ZoneCache) name(index int) string {
 		zoneCache.RUnlock()
 	}
 	if !ok { // last resort
-		name = uitoa(uint(index))
+		name = itoa.Uitoa(uint(index))
```

Comment:
nice catch. What about `dtoi` on line 250?

Reply:
That one IS defined here https://github.com/tinygo-org/tinygo/blob/dev/src/net/parse.go#L14